drivers/pwm/pwm_mcux_sctimer.c

```diff
@@ -31,13 +32,62 @@ struct pwm_mcux_sctimer_data {
 	sctimer_pwm_signal_param_t channel[CHANNEL_COUNT];
 };
 
+static int mcux_sctimer_pwm_init_device(
+	const struct pwm_mcux_sctimer_config *config)
+{
+	sctimer_config_t pwm_config;
+	status_t status;
+
+	SCTIMER_GetDefaultConfig(&pwm_config);
+	/* Divide the SCT clock by 8 */
```

Comment:
This comment is only accurate if `config->prescale = 8`, correct?

Reply:
I cannot tell since this is original code I extracted from the ``init`` function. Right, it might be confusing, will delete!